venom_core/execution/skills/input_skill.py

```
@@ -5,19 +5,47 @@
+    return module
+
+
+def _load_pyautogui():
+    """Import pyautogui without suppressing real application exits."""
+
+    if importlib.util.find_spec("tkinter") is None and "mouseinfo" not in sys.modules:
+        sys.modules["mouseinfo"] = _build_mouseinfo_stub()
+
+    try:  # pragma: no cover - zależne od środowiska testowego
+        import pyautogui  # type: ignore[import-untyped]
+    except Exception:  # pragma: no cover
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)